CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpC
 
 * Raised minimal PHP version to 7.1
 * Test helper classes upgraded to support PHPUnit 7/8
+* Added [experimental support](https://foshttpcache.readthedocs.io/en/latest/proxy-clients.html#fastly-client) for [fastly](https://www.fastly.com/)
```

Comment:
```suggestion
* Added [experimental support](https://foshttpcache.readthedocs.io/en/latest/proxy-clients.html#fastly-client) for [fastly](https://www.fastly.com/)
* There is no discovery package 2.0, drop it from composer.json
* Adapt for changes to NetworkException in httplug 2.1 for PSR-18 compatibility 
```

Reply:
Oh, fastly is mentioned on 2.8.0 though:

```
Added Fastly ProxyClient Adapter with ClearCapable, PurgeCapable, RefreshCapable, & TagCapable.
  Fastly is a CDN originally based on Varnish 2.x, so with many of the same capabilities like VCL and more
```

Reply:
hm, odd. it shows up in the git log of changes since 2.8.0 ... but yeah, will remove it again